// inventory.proto

syntax = "proto3";

import "book.proto";

// The InventoryService
service InventoryService {
  // Creates a new book.
  rpc CreateBook(Book) returns (CreateBookResponse) {}

  // Gets a book by its ISBN.
  rpc GetBook(BookRequest) returns (Book) {}
}

message BookResponse {
  Book book = 1;
}

message BookRequest {
  string isbn = 1;
}

message CreateBookResponse {
  string success = 1;
}

